torchdynamo/debug_utils.py

```diff
@@ -159,32 +176,70 @@ def inductor_fails(fx_g, args, check_str=None):
     return False
 
 
-def dump_to_minify(gm, args):
+def nvfuser_fails(fx_g, args, check_str=None):
+    from torch.fx.passes.backends.nvfuser import NvFuserBackend
+
+    nvfuser = NvFuserBackend()
+
+    try:
+        compile_mod = nvfuser(fx_g, args)
+        compile_mod = compile_mod(*args)
+    except Exception as e:
+        if check_str is not None and check_str not in repr(e):
```

Comment:
Nit: Maybe add a comment on the API, It's not readily obvious that check str can result in fails returning a false on a mismatch. It almost feels like it maybe should be two APIS, fails/fails_with_message

Reply:
Yeah this API is not really that great haha.